code/modules/power/smes.dm

```diff
@@ -424,7 +424,7 @@
 	log_smes()
 
 /obj/machinery/power/smes/engineering
-	charge = 37.5 * STANDARD_CELL_CHARGE // Engineering starts with some charge for singulo //sorry little one, singulo as engine is gone
+	charge = 50 * STANDARD_CELL_CHARGE // Engineering starts with some charge for singulo //sorry little one, singulo as engine is gone
```

Comment:
Does charge variable actually work.

Upon initialization the machine calls RefreshParts() and that proc will override this value based on the cells charge so maybe you should actually adjust the charge of each individual cell such that the sum of their values equals this?

Reply:
It only does that if the machine starts with no charge.